libraries/iterate/README.md

```diff
@@ -17,7 +17,7 @@ i = new Iterate({
   maxIterationTime: 10000,
   watchdogTime: 5000,
   waitTime: 2000,
-  handler: async (watchdog, state) => {
+  handler: async watchdog => {
```

Comment:
I like this change a lot. I'm just surprised nothing in any service used `state`.

Reply:
There are only four uses of this library, and three of them (in queue) are cookie-cutter)..